### ISSUE TITLE
Copying maps to steamapps\common\Synergy\synergy does not apply fixes

### ISSUE THREAD

**Issue by Bartolucci** [yarik2720/Synergy-SM#37]
I have been testing the fixes for HL2 and none of the fixes are working.  After reading the README this appears to be the only step needed to apply these fixes.  Am I missing something or is there a console command I have to run?

**Comment by yarik2720** [comment]
Are all *.edt files are placed in the maps folder?
Make sure that you are running server from the same game directory

**Comment by Bartolucci** [comment]
Yeah, they're all in the map folder.  Does it make a difference that I'm not running the dedicated server?  I'm just running Synergy's client and hosting with a couple friends from there.  But I don't see why that would matter.

**Comment by Bartolucci** [comment]
Ok, I just tried a reinstall and it seems to work.  Breen's office is no longer jittery!  However, Alexa still doesn't spawn in Nova Prospekt's last map (she is needed to open some doors near the beginning of the map).

That would be map d2_prison_08.  I use "mp_antirush_disable 0" and restart the map by using "changelevel", but this does not work.  I've already played through this segment with my friends so it's no big deal, but I thought I would bring it to your attention.
